jpa/EAP7-1986_Hibernate6_2.adoc

```diff
@@ -0,0 +1,73 @@
+* https://github.com/hibernate/hibernate-orm/discussions/5104[Support mapping composite/struct types] via https://hibernate.atlassian.net/browse/HHH-15327[HHH-15327].
+* https://github.com/hibernate/hibernate-orm/discussions/5105[Common table expression] via https://hibernate.atlassian.net/browse/HHH-15328[HHH-15328].  https://en.wikipedia.org/wiki/Hierarchical_and_recursive_queries_in_SQL[Also see Hierarchical_and_recursive_queries_in_SQL].
+* https://hibernate.atlassian.net/browse/HHH-15175[Remove support for database versions that are unsupported by vendors via HHH-15175.]
+
+Hibernate Search needs to be upgraded to 6.1.8+ https://hibernate.atlassian.net/browse/HSEARCH-4735[via HSEARCH-4735 for testing with Hibernate ORM 6.2.]
+
+The new libraries were added for Jakarta EE 10 compatibility (https://jakarta.ee/compatibility/certification/10).
+
+Migration notes for transition from Hibernate ORM 5.3 to 6.1 can be viewed via, these pages will be used as input for the EAP 8 documentation updates:
+
+* https://github.com/hibernate/hibernate-orm/blob/5.4/migration-guide.adoc
+* https://github.com/hibernate/hibernate-orm/blob/5.5/migration-guide.adoc
+* https://github.com/hibernate/hibernate-orm/blob/5.6/migration-guide.adoc
+* https://github.com/hibernate/hibernate-orm/blob/6.0/migration-guide.adoc
+* https://github.com/hibernate/hibernate-orm/blob/6.1/migration-guide.adoc
+* https://github.com/hibernate/hibernate-orm/blob/main/migration-guide.adoc (work in progress towards producing https://github.com/hibernate/hibernate-orm/blob/6.2/migration-guide.adoc).
+
+We expect that for https://issues.redhat.com/browse/EAP7-1985[EAP7-1985 we will collect Telemetry] from Hibernate ORM 6.2 via the JipiJapa Persistence integration layer to be consumed by a common client.
+
+== Issue Metadata
+
+=== Dev Contacts
+
+* mailto:{Scott Marlow}[{smarlow@redhat.com}] JPA / WildFly integration lead
+* mailto:{Paul Ferraro}[{pferraro@redhat.com}] WildFly clustering lead
+* mailto:{Steve Ebersole}[{steve@hibernate.org}] Hibernate ORM, project lead
+* mailto:{Sanne Grinovero}[{sanne@hibernate.org}] Hibernate team lead
+
+
+=== Other Interested Projects
+
+Infinispan project will support the Hibernate ORM 6 second level cache interfaces (SPI) which are compatible with the Hibernate ORM 5.3 second level cache interfaces (SPI) so that EAP 8 users can use a Persistence 2nd Level Cache.
+
+== Requirements
+
+Hibernate ORM 6.2+, Hibernate Search 6.1.8+.
+
+== Test Plan
+
+The Jakarta EE 10 TCKs must be passed with the upgraded Hibernate ORM version.
+
+* The Hibernate ORM testsuite must be passed against each supported database listed in https://issues.redhat.com/browse/EAP7-1820
+* Note that (SAP) Sybase SQL Server cannot be tested currently in the community since there is no (community) JDBC driver for Sybase SQL Server (Adaptive Server Enterprise) so this should be tested as soon as possible.
```

Comment:
We are currently using the jTDS driver for testing which is not maintained anymore: https://github.com/hibernate/hibernate-orm/blob/main/settings.gradle#L203

Reply:
Wording updated to mention jTDS driver.